packages/web/app/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,104 @@
+import {
```

Comment:
Can you explain what this file is doing? I don't see it referenced anywhere.

Reply:
was testing something with it but deleted it !